bin/bootstrap-harmony

```diff
@@ -22,30 +22,30 @@ else
   ./bin/create-dotenv
 
   # remove the empty OUATH env vars already in the file
-  awk -v n=3 'NR==FNR{total=NR;next} FNR==total-n+1{exit} 1' .env .env > env.tmp && mv env.tmp .env
-
-cat <<PROMPT_EDL
-Running Harmony requires creating an Earthdata Login (EDL) application. If you need instructions for this or do not understand what this means, please contact the Harmony team for help. Once you have your EDL application created, please enter the credentials as prompted.
-
-PROMPT_EDL
-
-  # prompt for the users EDL app credentials
-  read -p "EDL Application Client ID: " OAUTH_CLIENT_ID
-  echo ""
-  read -s -p "EDL Application UID: " OAUTH_UID
-  echo ""
-  read -s -p "EDL Application Password: " OAUTH_PASSWORD
-  echo ""
-
-cat << EOF >> .env
-OAUTH_CLIENT_ID=$OAUTH_CLIENT_ID
-OAUTH_UID=$OAUTH_UID
-OAUTH_PASSWORD=$OAUTH_PASSWORD
-EOF
-
-  if [[ $(uname) == 'Linux' ]]; then
-    echo "KUBE_CONTEXT=minikube" >> .env
-  fi
+  # awk -v n=3 'NR==FNR{total=NR;next} FNR==total-n+1{exit} 1' .env .env > env.tmp && mv env.tmp .env
```

Comment:
This is all commented out but it looks like it should be removed, since it's moved to create-dotenv

Reply:
Thanks for spotting this. Fixed.